### PR TITLE
add basic UI

### PR DESCRIPTION
**Pull request by Abdulwhbhussain**
Project requirements
Watch the video with the UI example again (you have seen it while reading the Sneak Peek).
UI example
Your console app should behave in the same way as it is in the example.
Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following:
List all books.
List all people.
Create a person (teacher or student, not a plain Person).
Create a book.
Create a rental.
List all rentals for a given person id.
In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:
Present the user with a list of options to perform.
Lets users choose an option.
If needed, ask for parameters for the option.
Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    print 'Has parent permission? [Y/N]: '
+    permission = gets.chomp.upcase
+    permission = (permission == 'Y')
+    puts 'testing before student creation'
+    student = Student.new(name, age, permission)
+    puts 'testing after student creation'
+    @people.push(student)
+    puts 'Student created successfully'
+    puts ' '
+  end
```
Comment (Reem-lab) [inline app.rb]: - [ ] Great job working on the functionality so far but when I try to create a new student, I face this error with the wrong number of arguments (given 3, expected 1..2) so kindly check on this error and try to solve the problem to let the user be able to create a student ⭐🔥💯🙏🏻

- When I try to create a student  ⬇
image

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @Abdulwhbhussain, 👋🏻👋🏻

Great job so far! ✨⭐🤩✅
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- [ ] No Linters error ✅
- [ ] Followed Gitflow ⭐
- [ ] Great job working on the whole functionality f 💯✨
- [ ] Descriptive PR 🚀

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (Abdulwhbhussain) [reply app.rb]: Solved Bro!

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Abdulwhbhussain 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- All options are working like in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) provided for you :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**